src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,41 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../services/users';
+import { User } from '../../types/User';
+
+type Props = {
+  setHasTodoClick: (a: boolean) => void;
+  fillteredTodos: Todo[];
+  userId: number;
+  todoId: number;
+  loading: boolean;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  setHasTodoClick,
+  fillteredTodos,
+  userId,
+  todoId,
+  loading,
+}) => {
+  const [user, setUser] = useState<User | undefined>(undefined);
+
+  useEffect(() => {
+    getUser(userId).then(setUser);
+  }, [userId]);
+
+  function handleCloseButton() {
+    setHasTodoClick(false);
+  }
+
+  const selectTodo = fillteredTodos.find(todo => todo.id === todoId);
```

Comment:
The usage of 'fillteredTodos' is misspelled. It should be 'filteredTodos'. Ensure consistency with the corrected variable name.